packages/ui/src/Categories.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+import { Button } from "./shad/ui/button";
+import { useRecoilState } from "recoil";
+import { category, search } from "@repo/store";
+import { Input } from "./shad/ui/input";
+import { MagnifyingGlassIcon } from "@radix-ui/react-icons";
+
+export const Categories = () => {
+  function handleFilterButton(category: string) {
+    if (category == selectedCategory) {
+      setSelectedCategory("");
+    } else {
+      setSelectedCategory(category);
+    }
+  }
+  const categories = ["Fundamentals", "Advance", "Devops", "Web3", "DSA"];
```

Comment:
Should come from the DB

Reply:
@hkirat 
I have made changes in #148 can you please review it?